probe/tag/tagger.go

```diff
@@ -0,0 +1,25 @@
+package tag
+
+import "github.com/weaveworks/scope/report"
+
+// Tagger tags nodes with value-add node metadata.
+type Tagger interface {
+	Tag(r report.Report, ts report.TopologySelector, id string) report.NodeMetadata
```

Comment:
- do we ever tag anything other than a whole report?  
- is there any case where we might want to apply the same tagger to different topologies?